database/migrations/2020_10_11_095007_create__user_logins_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUserLoginsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_logins', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('user_agent');
```

Comment:
string for ip (I think `45` char length is correct for ipv6?)

Reply:
Sure.

Reply:
> Hi @g1dra !
> Cheers for the PR.
> For the most part looks like its there, although have flagged a few minor niggles, if you have a few minutes to work through them.
> 
> Btw, having got the app set up - any initial feedback on the process / initial experince?

Nav needs to be changed its kinda confusing how it hides/changes. Project seams very interesting. UI kinda needs more styling.